tests/tests_fabric/parity/utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import torch
+
+
+def is_state_dict_equal(state0, state1):
+    # TODO: This should be torch.equal, but MPS does not yet support this operation (torch 1.12)
```

Comment:
You can make this self documenting by doing:


```python
eq_fn = torch.close if _TORCH_GREATER_EQUAL_1_12 else torch.allclose
```

Reply:
This was already there before, I just moved the code.